.circleci/config.yml

```diff
@@ -201,6 +212,11 @@ jobs:
       - restore_cache:
           keys:
             - v1-model_cache-{{ checksum "setup.py" }}
+
+      # Restore testmondata
+      - restore_cache:
```

Comment:
Not something critical, shall we only restore the cache for the open PR long integration cached test?

Reply:
Actually I tried a different way to simplify the circle ci config. Please take a look.

Reply:
Oh, I meant it is currently restoring the cache for any test. Is that expected? Do we want to rerun everything for `staging` and `master`? Or we should run only affected as well?